themes/xltdoc/layouts/shortcodes/TODO.html

```diff
@@ -1 +1 @@
-<!--<span style="color:magenta; font-weight:bold; background-color: yellow">{{ .Inner }}</span>-->
\ No newline at end of file
+{{ if .Inner }}<span class="todo" {{ with .Get "comment"}} title="{{.}}"{{ end }}>{{ .Inner }}</span>{{ else }}<span class="todo" {{ with .Get "comment"}} title="{{.}}"{{ end }}></span>{{ end }}
```

Comment:
Let's talk about this rework tomorrow. My intention had been that TO DOs are visible (and searchable) for doc developers in md code, but completely invisible in the public documentation. Not sure whether this makes more sense.

Reply:
I am pretty sure these internal TODOs will be overlooked mostly when not visible someplace. Don't see a problem with sensible placeholder TODOs. We are also looking for help, maybe that helps. We can also display: none these if needed. Let's keep that visible as long as we know that there is still enough to do. Rest is a simple comment if needed.

Reply:
The new comment mode enables less intrusive TODOs.